gcp/modules/k8s-snapshots/values.yaml

```diff
@@ -0,0 +1 @@
+useClaimName: false
```

Comment:
Why is this needed? We use `true` on the AWS side, which produces much better Snapshot names.

Reply:
Related to K8s-managed vols vs Terraform-managed.